setup.py

```diff
@@ -42,4 +64,5 @@ def convert(filename, fmt):
     long_description=convert('README.md', 'rst'),
     include_package_data=True,
     zip_safe=False,
+    data_files=create_mo_files(),
```

Comment:
can you point me on any relevant Django project which handles PO-files like this. That's the first time I see this and I don't see any relevant advantage.

Reply:
it was to avoid having launched "django-admin compilemessages" after installation, other project like aldryn-forms provide the .mo in the git and the package.